Refresh.Interfaces.Game/Endpoints/Levels/LevelEndpoints.cs

```diff
@@ -113,6 +115,56 @@ public class LevelEndpoints : EndpointGroup
         return this.GetLevels(context, database, categories, overrideService, user, token, dataContext, route);
     }
 
+    // Route example: /lbp/slots/like/user/276&pageStart=1&pageSize=30
+    // The syntax error in the query params (& instead of ?) makes Bunkum include them as part of the ID route param
+    [GameEndpoint("slots/like/{slotType}/{id}", ContentType.Xml)]
+    [MinimumRole(GameUserRole.Restricted)]
+    public Response GetLevelsLikeLevel(RequestContext context, DataContext dataContext, GameUser user, string slotType, string id)
```

Comment:
This seems useful. In the future we could move this logic to the database and have a recommendation algorithm on level pages on the website.